### PR TITLE
Building with Backgrounds and Gradients - Features branch - complete site

### PR DESCRIPTION
**Pull request by canriquez**
Please review the code for change requests/approval.

The goal is to make a clone of old an old version of the Apple website webpage.
- Use semantics
- Build a webpage using images as a background
- Add gradients to elements. 
- Elements aligned with Flex

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            <h1><a href="#"><img src="/assets/img/whatwill.png" alt="What will your verse be?"></a></h1>
+            <h2><a href="#"><img src="/assets/img/everyone1.png" alt="Everyone has something to share. See how people are using iPad in their unique ways to add to the world’s story."></a></h2>
+            <div><a href="#"><img src="/assets/img/watch2.png" alt="Watch the “Your Verse” film and explore the stories"></a></div>
+        </header>
+        <aside>
+            <ul class="aside-blocks">
+                <li class="iphone5s"><a href=""><span></span></a></li>
+                <li class="iphone5c"><a href=""><span></span></a></li>
+                <li class="happy"><a href=""><span></span></a></li>
+                <li class="supplier"><a href=""><span></span></a></li>
```
Comment (pelzolga123) [inline index.html]: `#` is missing inside `<a>` tags.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+            </ul>
+        </aside>
+        <footer>
+            <div class="footer-content">
+                <div class="footer-text shop-text">Shop the
+                            <a href="">Apple Online Store</a>
+                            (1-800-MY-APPLE), visit an 
+                            <a href="">Apple Retail Store</a>, or find a 
+                            <a href="">reseller</a>
```
Comment (pelzolga123) [inline index.html]: `#` is missing inside `<a>` tags.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+                <div class="footer-text">
+                            <div>Copyright © 2014 Apple Inc. All rights reserved.</div>
+                            <ul class="terms-privacy">
+                                <li><a href="">Terms of Use</a></li>
```
Comment (pelzolga123) [inline index.html]: `#` is missing inside `<a>` tag.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+                                <li id="privacy"><a href="">Privacy Policy</a></li>
```
Comment (pelzolga123) [inline index.html]: `#` is missing inside `<a>` tag.

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Links to images need to be fixed, some `<a>` tags are missing `#`.

**Comment by pelzolga123** [comment]
## Hi Carlos!  
### Your did a great job :+1: !
### Kudos for descriptive pull request and clean code! :clap: 

Only few things need to be changed:
* [x] Images are not displaying correctly:
image

It can be fixed if you change your image url's:
* From `/assets/img/image_name.png` to `../img/image_name.png`  in main.css
* From `/assets/img/image_name.png` to `./assets/img/image_name.png`  in index.html
* [x] Some links are missing `#`. 

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by canriquez** [comment]
Thanks Olga, all changes are now implemented. This was working on my local server, but I forgot to check on raw.githack.com. I believe now is fixed. thanks!

**Review by pelzolga123 — APPROVED** [review]
Approved.

**Comment by pelzolga123** [comment]
## Great  work, Congratulation :+1: Project Approved.
You did all changes required previously. Go ahead and merge to master.
_Goodluck in your next project_ *Happy coding*

**Pull request merged**